feature-group-definitions/column-breaks.yml

```diff
@@ -0,0 +1,15 @@
+name: Column breaks
+description: In columnar layouts (created by the `columns` or `column-count` CSS properties), the `break-after`, `break-before`, `break-inside` properties control where columns start or end.
```

Comment:
This one does something a little funny, setting the context before describing the thing. It does make its description distinct from the other breaking feature though, which I think makes this in the spirit of the rule if not the letter of it.

Reply:
Makes sense to me!